search-commons/src/test/java/no/unit/nva/search2/OpenSearchSwsClientTest.java

```diff
@@ -18,35 +25,38 @@
 import java.util.Collections;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
-
-import static no.unit.nva.indexing.testutils.MockedJwtProvider.setupMockedCachedJwtProvider;
-import static nva.commons.core.ioutils.IoUtils.stringFromResources;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import javax.net.ssl.SSLSession;
+import no.unit.nva.search2.model.OpenSearchQuery;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 class OpenSearchSwsClientTest {
 
-    private OpenSearchSwsClient openSearchSwsClient;
-    private static final String MEDIA_TYPE = "application/json";
     public static final String SAMPLE_OPENSEARCH_RESPONSE_RESPONSE_EXPORT
         = "sample_opensearch_response.json";
+    private OpenSearchSwsClient openSearchSwsClient;
+
+    public final Logger logger = LoggerFactory.getLogger(OpenSearchSwsClientTest.class);
 
     @BeforeEach
     public void setUp() throws IOException, InterruptedException {
+
         var httpClient = mock(HttpClient.class);
-        var mockedResponse = mockedHttpResponse();
         var cachedJwtProvider = setupMockedCachedJwtProvider();
-
-        when(httpClient.send(any(), any())).thenReturn(mockedResponse);
-
         openSearchSwsClient = new OpenSearchSwsClient(cachedJwtProvider, httpClient);
+        when(httpClient.send(any(), any()))
+            .thenReturn(mockedHttpResponse(SAMPLE_OPENSEARCH_RESPONSE_RESPONSE_EXPORT));
     }
 
     @NotNull
-    private HttpResponse<Object> mockedHttpResponse() {
+    public static HttpResponse<Object> mockedHttpResponse(String filename) {
```

Comment:
why is a test class method public static?

Reply:
Good question, I have two testclasses for OpenSearchSws, one with an empty resultset and another with results.
Since I use mockedHttpResponse for both, I made it public static... 
It will be deleted when OpenSearchAws has been testet and works